doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -847,6 +833,43 @@ Modem libraries
       Refer to the :ref:`migration guide <migration_2.8>` for more details.
     * The :c:enum:`lte_lc_reduced_mobility_mode` type has been deprecated.
       Refer to the :ref:`migration guide <migration_2.8>` for more details.
+    * The library to be reorganized into modules that are enabled via their respective Kconfig options.
+      By default, the library now enables only the core features related to the network connectivity.
+      This reorganization reduces flash memory consumption for applications that only use the core features of the library related to network connectivity.
+      For more information on how to update your project, see the :ref:`migration guide <migration_2.8_required>`.
+
+      * Added the new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable the Connection Parameters Evaluation module.
```

Comment:
```suggestion
      * The :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable the Connection Parameters Evaluation module.
```
Now that we have the mention about the reorg of the library, I think we could have these new Kconfig options under Added, after all. Without the "Added the new" like in my suggestion here.